js/state/frameStateUtil.js

```diff
@@ -233,27 +213,63 @@ function getPartition (frameOpts) {
   }
   return partition
 }
+
+function cloneFrame (frameOpts, guestInstanceId) {
+  const cloneableAttributes = [
+    'audioMuted',
+    'canGoBack',
+    'canGoForward',
+    'icon',
+    'title',
+    'isPrivate',
+    'partitionNumber',
+    'themeColor',
+    'computedThemeColor'
```

Comment:
what about `provisionalLocation` and maybe `src` and `location`.
provisionalLocation I can see maybe being useful on a about:certerror type of page.